src/ArborX_LinearBVH.hpp

```diff
@@ -294,9 +307,11 @@ void BoundingVolumeHierarchy<MemorySpace, Enable>::query(
   {
     Kokkos::Profiling::pushRegion(profiling_prefix + "::compute_permutation");
     using DeviceType = Kokkos::Device<ExecutionSpace, MemorySpace>;
+    Box bbox{};
+    Details::expand(bbox, bounds());
```

Comment:
I think it would make sense to store both `bounding_volume _bounds` and `Box _scene_box`.